pkg/e2e/main_test.go

```diff
@@ -0,0 +1,37 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package e2e
+
+import (
+	"os"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+)
+
+func TestMain(m *testing.M) {
+	logrus.Info("Running tests on COMPOSE_STANDALONE_MODE=true")
+	_ = os.Setenv(composeStandaloneEnvVar, "true")
```

Comment:
Can’t we do this using a command line flag and a (global) variable instead?

Reply:
My second option was to run the tests command line 2 times. One for each configuration of the envvar.
I'm not sure if I understood what you mean by using a global variable for this.

Reply:
Why do you need an env var at all? Is this not the same process?

Reply:
Ahhh... Finally got what you mean. The idea was to have it also configurable from outside of this `TestMain`.

With the envvar we can run just one specific test. And when running like this, `TestMain` doesn't run. So the envvar is a solution to be able to still pass this info to the test.

Reply:
But can’t we use a command line flag instead?
Sorry it’s just I don’t like env vars much, they tend to magically configure (break) things, a bit like global vars on steroids 😀

Reply:
The only alternative I see is to pass a build tag when running the test to choose in between tagged files. So we can choose which file to use, so we can use that conditionally depending on the tag.... But that really looks like over engineering and gets to the same problematic in the end...